java/vortex-spark/src/test/java/dev/vortex/spark/VortexScanTest.java

```diff
@@ -24,6 +25,10 @@
 final class VortexScanTest {
     private static final Path TPCH_ROOT = Paths.get("/Volumes/Code/vortex/bench-vortex/data/tpch/1/vortex_compressed");
 
+    static {
+        NativeLogging.initLogging(NativeLogging.DEBUG);
+    }
```

Comment:
Adding this gives us logging from the Rust lib, including the log of async task timings:

<img width="922" alt="image" src="https://github.com/user-attachments/assets/74507dae-2063-493b-b537-cc41bbc76c62" />